msdt-2/complexing.py

```diff
@@ -0,0 +1,61 @@
+import json
+import numpy as np
+import cv2 as cv
+
+errors_mas = np.array([[1,2,3],[4,5,6],[7,8,9]])

```

Comment:
Матрицы лучше оформлять таким образом
```suggestion
errors_mas = np.array(
    [
        [1,2,3],
        [4,5,6],
        [7,8,9]
    ]
)